app/assets/src/components/views/samples/SamplesView.jsx

```diff
@@ -34,31 +35,36 @@ class SamplesView extends React.Component {
 
     this.columns = [
       {
+        columnData: SAMPLE_TABLE_COLUMNS_V2["sample"],
```

Comment:
For DRY'ing this out, you could instead have a line looping through `this.columns` and adding `columnData: SAMPLE_TABLE_COLUMNS_V2[dataKey]`